drivers/android-driver/src/main/java/com/squareup/sqldelight/android/AndroidSqliteDriver.kt

```diff
@@ -162,21 +162,23 @@ class AndroidSqliteDriver private constructor(
     ) : this(schema, *emptyArray())
 
     override fun onCreate(db: SupportSQLiteDatabase) {
-      schema.create(AndroidSqliteDriver(openHelper = null, database = db, cacheSize = 1))
+      val driver = AndroidSqliteDriver(openHelper = null, database = db, cacheSize = 1)
+
+      schema.create(driver)
+      schema.migrate(driver, 0, schema.version)
```

Comment:
This does not seem correct. The schema should always represent the latest version, not the 0th version. No migrations should run on creation. I think the sample is just broken.

I allowed the tests to run because I'm hoping they fail on such a change.